pkg/util/lifted/lua/oslib_safe.go

```diff
@@ -64,7 +64,7 @@ func osTime(L *lua.LState) int {
 		year := getIntField(tbl, "year", -1)
 		isdst := getBoolField(tbl, "isdst", false)
 		t := time.Date(year, time.Month(month), day, hour, min, sec, 0, time.Local)
-		// TODO dst
+		// TODO: dst
```

Comment:
@jameszhangyukun I didn't remember what the plan is here. Can you help recall it? :)